### PR TITLE
Merge our feature branch including the test cases

### PR DESCRIPTION
**Pull request by davidauza-engineer**
Please let us know your comments about our work. Thanks in advance.

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--require spec_helper
```
Comment (wathigo) [inline .rspec]: Lets remove `--require spec_helper`. Its causing an error when we try to run the tests. We don't have `spec_helper` in our project

File: spec/enumerables_spec.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+# spec/enumerables_spec.rb
+
+require_relative '../main.rb'
```
Comment (wathigo) [inline spec/enumerables_spec.rb]: I think we have required the file wrongly. Let's change `require_relative '../main.rb'` to `require_relative '../bin/enumerable.rb'`.

**Review by wathigo — CHANGES_REQUESTED** [review]
### Hi @davidauza-engineer and @falguniislam  :wave:  
Your tests are not getting executed. This is due to the errors pointed out in the comments below :point_down: 
**[REQUIRED]** 
* Let's fix the issue pointed out in the comments to make sure that the tests executes successfully.

**After implementing the changes, submit another request for review.** 

_Happy Coding :computer:  
Reviewed by: **Simon Wathigo**_

File: .rspec (hunk 1 above)
Comment (davidauza-engineer) [reply .rspec]: Done :) Thank you

File: spec/enumerables_spec.rb (hunk 2 above)
Comment (davidauza-engineer) [reply spec/enumerables_spec.rb]: Done, thank you so much

**Comment by davidauza-engineer** [comment]
All the changes were made. Please let us know your comments 👍

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: . Your project is approved :heavy_check_mark:  and you may merge to development :+1: :+1: 

_Keep up the good work_ :muscle: 
**Ahmed Mahfoudh**

**Comment by davidauza-engineer** [comment]
Thank you so much guys @wathigo && @stratospherique  👍

**Pull request merged**